src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/CreateSiteFilteringFiles.java

```diff
@@ -0,0 +1,110 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import io.grpc.netty.shaded.io.netty.util.internal.StringUtil;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ShortVariantDiscoveryProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.engine.VariantWalker;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.variantdb.*;
+import org.broadinstitute.hellbender.tools.variantdb.IngestConstants;
+import org.broadinstitute.hellbender.utils.tsv.SimpleXSVWriter;
+
+import java.util.*;
+import java.io.File;
+import java.io.IOException;
+
+@CommandLineProgramProperties(
+        summary = "Create files for ingesting Site-level filtering data into BigQuery",
+        oneLineSummary = "Site-level filtering Data Ingest tool for GVS",
+        programGroup = ShortVariantDiscoveryProgramGroup.class,
+        omitFromCommandLine = true
+)
+public final class CreateSiteFilteringFiles extends VariantWalker {
+    static final Logger logger = LogManager.getLogger(CreateVariantIngestFiles.class);
```

Comment:
should this use CreateSiteFilteringFiles.class instead?

Reply:
yup